### PR TITLE
Siri & Tobias - Labyrinth

### PR DESCRIPTION
**Pull request by sirimatsdotter**
Link to my deployed project: https://reverent-blackwell-c57bf4.netlify.app/

My reflections on how this week's project turned out: We struggled a lot this week and got stuck several times. But we were quite happy in the end and manage to fulfill all blue and some red goals. So we are happy that we were able to hand something decent in :)

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

**Review by idasylven — APPROVED** [review]
We find your project VERY visually pleasing! 🌟🌟🌟 Great use of images to correspond with the instructions and to sent the mood of the game, we were very impressed with how you managed to render them 📸  

We could´nt tell whatsoever that you have been struggling. We understand you ambition with the video clips on each page - fingers crossed you´ll get that to work too 🤞 

From UX perspective having font color as white some of the text blends into the background imagery and makes it hard to read. This occurs mainly in coordinate 0.0 (mobile view) and 0.3 and 1.0 (desktop view). 

Overall the code is very well written, neat, structured and easy to follow! 👌👌👌  
GREAT JOB 🎉

File: code/src/components/Labyrinth.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      ? imgURL_1_0
+      : props.coordinates === '0,3'
+      ? imgURL_0_3
+      : props.coordinates === '0,2'
+      ? imgURL_0_2
+      : props.coordinates === '0,1'
+      ? imgURL_0_1
+      : props.coordinates === '0,0'
+      ? imgURL_0_0
+      : imgURL_start};
```
Comment (KarolCislowski) [inline code/src/components/Labyrinth.js:44]: if you name them `imgURL-1,3` then you can skip all the logic with `imgURL-${props.coordinates}` 🤓

File: code/src/components/Labyrinth.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    </Video>*/
+
+
+    /* const Video = styled.video`
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  min-width: 100%;
+  min-height: 100%;
+`
```
Comment (KarolCislowski) [inline code/src/components/Labyrinth.js:105]: remember to clean your code before sending the pull request.

**Review by KarolCislowski — APPROVED** [review]
Great job 🥇

**Pull request closed without merging**